src/workloads/execution/CumulativeWindows.yml

```diff
@@ -0,0 +1,155 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    # Small collection size to keep from spilling to disk during sorts.
```

Comment:
This is true, but also the window function stage itself can now spill. Given the way the memory tracking is currently working, we'll use approx double the size of each document as well as the state in the window (which for these type of windows should be minimal). So I think we're still good on not spilling?

Reply:
I can be more specific, but yes for these windows the sort is the only operation that should introduce the potential for spilling.